src/renderer/shared/hocs/withToast.js

```diff
@@ -11,11 +11,11 @@ function withToast({ type, propName = 'showToast', ...options } = {}) {
 }
 
 export function withSuccessToast({ propName = 'showSuccessToast', ...options } = {}) {
-  return withToast({ ...options, propName, type: TYPE_ERROR });
+  return withToast({ ...options, propName, type: TYPE_SUCCESS });
 }
 
-export function withErrorToast({ propName = 'showErrorToast', ...options } = {}) {
-  return withToast({ ...options, propName, type: TYPE_SUCCESS });
+export function withErrorToast({ propName = 'showErrorToast', autoDismiss = false, ...options } = {}) {
```

Comment:
Why is this defaulted to false - I sits there forever even when browsing to other pages :P

Reply:
An error message seemed important enough to leave up in case someone is multitasking so they don't miss it.  I agree that it doesn't make sense for it to remain on the screen when navigating away from a page though, I'll look into clearing flashes when the route changes.

Reply:
@DalderupMaurice I added another commit here so that any open toasts are automatically close when the route changes.